### ISSUE TITLE
Output no longer loading for code cells that generate errors

### ISSUE THREAD

**Issue by aparente-nurix** [microsoft/vscode#139545]
Issue Type: <b>Bug</b>

I just updated VScode insiders and now when I run a cell that generates an error, I just get the red X at the bottom of the cell but no error message or any output appears making it impossible to debug. I can see the output from code cells that run without errors. 

Extension version: 1.0.4
VS Code version: Code - Insiders 1.64.0-insider (f67a8b753f55e452f38b61497662839c4691fb12, 2021-12-16T05:15:05.411Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No
Fetching remote diagnostics for 'ssh-remote+aparente-t7820' timed out.
Remote OS version: Linux x64 3.10.0-1062.18.1.el7.x86_64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.82GB (5.69GB free)|
|Process Argv|--crash-reporter-id 3ab5d30a-e57d-4c95-ad43-4ab531c2c975|
|Screen Reader|no|
|VM|0%|

Fetching remote diagnostics for 'ssh-remote+aparente-t7820' timed out.

|Item|Value|
|---|---|
|Remote|SSH: aparente-t7820|
|OS|Linux x64 3.10.0-1062.18.1.el7.x86_64|
|CPUs|Intel(R) Xeon(R) Gold 6136 CPU @ 3.00GHz (48 x 1200)|
|Memory (System)|125.35GB (16.24GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30258533
vsc_aa:30263845
pythonvspyt551cf:30291413
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
vscop804:30404766
vs360cf:30404996
vsclayoutctrt:30405800
py55gd98:30408621
vscexrecpromp3t2:30407759
azactmsalcf:30409288

```

</details>

**Comment by DonJayamanne** [comment]
Looks like we're having problems in VS Code loading the renderers

**Comment by wesley-cheek** [comment]
Seeing the same issue here.
![image](https://user-images.githubusercontent.com/66931618/147038723-57f0a60f-f6e5-47cb-b5aa-86534eb70dc2.png)

Issue Type: Bug

Open a Jupyter Interactive Window.
Do something that causes an error:

`1 / 0`

Output:

```
Error loading preloads:
Failed to fetch dynamically imported module: https://file+.vscode-resource.vscode-webview.net/c%3A/Users/104104/.vscode-insiders/extensions/ms-toolsai.jupyter-2022.1.1001605991/out/datascience-ui/errorRenderer/errorRenderer.js
```

Extension version: 1.0.4
VS Code version: Code - Insiders 1.64.0-insider (e8a26d3ebf5095d4324ad4bf794787540ed3b33e, 2021-12-21T05:15:30.501Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

<html>
<body>


CPUs | Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz (12 x 2712)
-- | --
GPU Status | 2d_canvas: enabled gpu_compositing: enabled multiple_raster_threads: enabled_on oop_rasterization: enabled opengl: enabled_on rasterization: enabled skia_renderer: enabled_on video_decode: enabled vulkan: disabled_off webgl: enabled webgl2: enabled
Load (avg) |  
Memory (System) | 15.61GB (3.32GB free)
Process Argv | --crash-reporter-id cb42f697-ad09-444c-b86d-8cd62a1d3e46
Screen Reader | no
VM | 0%


</body>
</html>
<html>
<body>


Extension | Author (truncated) | Version
-- | -- | --
monokai-charcoal-high-contrast | 74t | 3.4.0
better-comments | aar | 2.1.0
Bookmarks | ale | 13.2.2
gitlens | eam | 11.7.0
shell-format | fox | 7.2.2
vsc-python-indent | Kev | 1.14.2
select-highlight-minimap | mde | 0.1.0
git-graph | mhu | 1.30.0
vscode-docker | ms- | 1.18.0
python | ms- | 2022.0.1573923366-dev
vscode-pylance | ms- | 2021.12.3-pre.1
jupyter | ms- | 2022.1.1001605991
jupyter-keymap | ms- | 1.0.0
jupyter-renderers | ms- | 1.0.4
remote-containers | ms- | 0.209.6
cpptools | ms- | 1.7.1
autodocstring | njp | 0.5.4
vscode-python-typehint | njq | 1.4.1
vim | vsc | 1.21.10

<span style="color: rgb(255, 255, 255); font-family: &quot;Segoe WPC&quot;, &quot;Segoe UI&quot;, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(0, 0, 0); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">(1 theme extensions excluded)</span>
</body>
</html>
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30258533
pythonvspyt551cf:30291413
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
vscop804cf:30404767
vscop453:30404998
vsclayoutctrc:30405799
py55gd98:30408621
vscexrecpromp3c:30407757
azactmsal:30409287

**Comment by matrix303** [comment]
Yes, seeing the same issue here. I have noticed that to get a detailed output, it is possible to click on the </> and select the built-in version to get the old error output. 

I have reproduced this on remote and on local.


<img width="1210" alt="Screen Shot 2021-12-22 at 9 20 31 PM" src="https://user-images.githubusercontent.com/10987493/147177971-bbb83265-8f9b-44d3-bd13-45fa90553c0e.png">
<img width="600" alt="Screen Shot 2021-12-22 at 9 21 09 PM" src="https://user-images.githubusercontent.com/10987493/147177981-10e19477-f4a5-4b74-9af5-619dc62372d3.png">
<img width="1188" alt="Screen Shot 2021-12-22 at 9 21 16 PM" src="https://user-images.githubusercontent.com/10987493/147177988-30584db7-b7a7-4253-b60a-848e2ac1ac8e.png">



Vscode info:
Version: 1.64.0-insider
Commit: 7b9e5c32a053669d4e72f998e4fa217090c32f59
Date: 2021-12-22T05:14:43.841Z (21 hrs ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.2.0


Jupyter v2022.1.1001605991
Jupyter Notebook Renderers v1.0.4
Python v2021.12.1559732655
